scripts/npmPublish.sh

```diff
@@ -1,10 +1,33 @@
 #!/usr/bin/env bash
 
-cd $(dirname $0)/..
+# Publishes TSLint to NPM
+# This script must be run with yarn: "yarn run publish:local gitTagToPublish"
+# A user running this script must have Palantir NPM organization credentials
+
 set -e
 
-echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
-chmod 0600 .npmrc
+if [[ $# -ne 1 ]]; then
+    echo "usage: yarn run publish:local gitTagToPublish"
```

Comment:
can't we grab this from `echo 'console.log("./package.json").version' | node` or something similar?